src/themes/teams-high-contrast/components/Divider/dividerVariables.ts

```diff
@@ -2,7 +2,9 @@ import { DividerVariables } from '../../../teams/components/Divider/dividerVaria
 import { Partial } from 'types/utils'
 
 export default (siteVars: any): Partial<DividerVariables> => ({
+  colors: {
+    primary: siteVars.white,
+  },
```

Comment:
No `secondary` here?

Reply:
Yes, it was never supported in `Divider` :(

Reply:
Got i! :\